app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::CommentsController < ApplicationController
+  def index
+    post = Post.find(params[:post_id])
+    comments = post.comments
+    render json: comments
+  end
+
+  def create
+    post = Post.find(params[:post_id])
+
+    json_request = JSON.parse(request.body.read)
+    text = json_request['text']
+
+    author = post.author
+
+    comment = post.comment.new(text:, author:)
+
+    return unless comment.save
+
+    render json: comment
```

Comment:
- [ ] It doesnt going to return anything when any error occur , i suggest handle the error part also,
```
 comment = post.comments.new(text: text, author: author) # Use 'comments' for association
  if comment.save
    render json: comment, status: :created
  else
    render json: { errors: comment.errors.full_messages }, status: :unprocessable_entity
  end
  ```